.gitignore

```diff
@@ -1,95 +1,5 @@
+## Dist directory
+dist
 
```

Comment:
I think using this default .gitignore file would help save time in ignoring other files in the future

Reply:
Yes, it will. Thanks, @Mcdavid95